xml/storage_lvm.xml

```diff
@@ -1647,9 +1647,10 @@ sudo lvreduce /dev/LOCAL/DATA</screen>
     <para>
      To check the exact behavior, run the <command>lvmconfig -l | grep
      global_filter_compat</command> command. If the result shows that the
-     <literal>global_filter_compat</literal> option exists and set to
+     <literal>global_filter_compat</literal> option exists and is set to
      <literal>0</literal>, the behavior described above applies. If the option
-     does not exist or set to <literal>1</literal>, the behavior remains unchanged.
+     does not exist or is set to <literal>1</literal>, the behavior remains
+     unchanged.
```

Comment:
Both the `set` v/ `is set` changes are potato/tomahto changes again. No idea what they're good for.